src/config/config.ts

```diff
@@ -9,7 +9,7 @@ export interface IConfig {
 }
 
 export const config: IConfig = {
-    port: +process.env.PORT || 3000,
+    port: +process.env.PORT || 5000,
```

Comment:
Dobry pomysł ze zmianą portu, ale to zmień też w `example.env`

Reply:
OK